### ISSUE TITLE
Unable to install with Laravel 8.*

### ISSUE THREAD

**Issue by nocodelab** [rollbar/rollbar-php-laravel#122]
Hello,
I'm unable to install the package in a specific project based on Laravel 8 (latest release).

It looks the package is not compatibile with **psr/log 2.0+**

<img width="1792" alt="Screenshot 2022-01-17 at 13 06 00" src="https://user-images.githubusercontent.com/4540118/149766670-55be255a-1ea7-4612-a44b-3a6e0585ac1b.png">

I'm not sure that downgrading psr/log is a good idea.
What do you recommend?

**Comment by xtfer** [comment]
It's not optimal, but it does work downgraded.

**Comment by gaetan-hexadog** [comment]
same issue when installing on Laravel 9
```shell
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - rollbar/rollbar-laravel v7.1.0 requires rollbar/rollbar ^2 -> satisfiable by rollbar/rollbar[v2.0.0, v2.1.0].
    - rollbar/rollbar-laravel v7.1.0-RC1 requires illuminate/support ^6.0|^7.0|^8.0 -> found illuminate/support[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - rollbar/rollbar[v2.0.0, ..., v2.1.0] require psr/log ^1 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires rollbar/rollbar-laravel ^7.1 -> satisfiable by rollbar/rollbar-laravel[v7.1.0-RC1, v7.1.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require rollbar/rollbar-laravel:*" to figure out if any version is installable, or "composer require rollbar/rollbar-laravel:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

Why should we downgrade psr/log ?

**Comment by danielmorell** [comment]
I am working on upgrading `rollbar/rollbar-laravel` to `rollbar/rollbar` v3 which already supports `psr/log` v2. This is my top priority at the moment.

**Comment by cyrusradfar** [comment]
Thank you @danielmorell

**Comment by danielmorell** [comment]
This should be resolved with the release of v7.2.
